.gitmodules

```diff
@@ -898,3 +898,6 @@
 [submodule "vendor/grammars/atom-language-nextflow"]
 	path = vendor/grammars/atom-language-nextflow
 	url = https://github.com/nextflow-io/atom-language-nextflow
+[submodule "vendor/grammars/conllu-linguist-grammar"]
+	path = vendor/grammars/conllu-linguist-grammar
+	url = https://github.com/odanoburu/conllu-linguist-grammar
```

Comment:
I'm actually impressed this works with Linguist, but nevertheless, you might want to change the extension of your grammar file to `.json` ;)

Reply:
Yup, it's definitely best to change this to `.json` just to be safe.

Reply:
will do!